topics/jupyter-notebook/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Jupyter Developers
+display_name: Jupyter Notebook
+github_url: https://github.com/jupyter/notebook
+logo: jupyter-notebook.png
+released: December 2011
+short_description: Jupyter Notebooks are documents that contain live code, visualizations, and narrative text.
+topic: jupyter-notebook
+url: https://jupyter.org/
+---
+
+Jupyter notebooks are documents which allow for creating and sharing live code, equations, visualizations and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more. 
```

Comment:
What do you think of "that" instead of "which" in "documents which allow"?

What about including this line from the notebook repository? "Jupyter notebook is a language-agnostic HTML notebook application for Project Jupyter."

Can you add a comma after "visualizations" in "equations, visualizations and narrative text"?

Reply:
Thanks for the feedback! I've (mostly) addressed these, let me know what you think about where I introduced the line about the notebook application (as opposed to the files).